### PR TITLE
Translation of Contact

### PR DESCRIPTION
**Pull request by alexjondiaz**
I am unsure on how to translate names, such as Näringslivsgruppen, and have named it "industry group" based on earlier translations. As well as "head of" for "-ansvarig".

### CODE REVIEW

**Comment by karlek** [comment]
I think this has also been an issue in the past. We've also talked about making a dictionary for having consistent and standardized translations, @bullfest showed me a list a few years ago if my memory right. Also found this KTH dictionary[1], which mainly focuses on academic terms, but at least it's something.

\[1\]: https://www.kth.se/social/group/sprakkommitten/page/kths-svensk-engelska-ordbok/

**Comment by bullfest** [comment]
Yea, there's a semi-official translation document. I believe @avelinpv is the one that maintains it at the moment.

**Comment by LudwikJaniuk** [comment]
This is still better than nothing so I'm merging it.

File: en/contact/body.md [hunk 1]
```diff
@@ -1,63 +1,60 @@
...
-## Styrelsen
+## The board
 
-För frågor om sektionens organisation bör du kontakta Datasektionens styrelse,
-D-rektoratet, på adressen [drek@d.kth.se](mailto:drek@d.kth.se). Du kan
-även kontakta Datasektionens Ordförande direkt:
+For questions regarding the chapter's organisation, contact the board at [drek@d.kth.se](mailto:drek@d.kth.se). You can also contact the head of the chapter directly at:
 
-**Ordförande**<br />
+**Head of the Computer Science Chapter**<br />
```
Comment (bullfest) [inline en/contact/body.md]: Chapter president

File: en/contact/body.md [hunk 2]
```diff
@@ -1,63 +1,60 @@
...
 Alexander Viklund<br />
-E-post: [ordf@d.kth.se](mailto:ordf@d.kth.se)<br />
+E-mail: [ordf@d.kth.se](mailto:ordf@d.kth.se)<br />
 076-015 61 42
 
-## Sektionsinformation
+## Chapter details
 
-**Postadress**
+**Mailing adress**
```
Comment (bullfest) [inline en/contact/body.md]: Postal address

File: en/contact/body.md [hunk 3]
```diff
@@ -1,63 +1,60 @@
...
 
 Konglig Datasektionen<br />
 Drottning Kristinas väg 15-19<br />
-Fack vid THS<br />
+Fack by THS<br />
 100 44 Stockholm
 
-**Besöksadress**
+**Visiting adress**
```
Comment (bullfest) [inline en/contact/body.md]: 2 d i address

File: en/contact/body.md [hunk 4]
```diff
@@ -1,63 +1,60 @@
...
+## The board
 
-För frågor om sektionens organisation bör du kontakta Datasektionens styrelse,
-D-rektoratet, på adressen [drek@d.kth.se](mailto:drek@d.kth.se). Du kan
-även kontakta Datasektionens Ordförande direkt:
+For questions regarding the chapter's organisation, contact the board at [drek@d.kth.se](mailto:drek@d.kth.se). You can also contact the head of the chapter directly at:
 
-**Ordförande**<br />
+**Head of the Computer Science Chapter**<br />
 Alexander Viklund<br />
```
Comment (bullfest) [inline en/contact/body.md]: Borde uppdateras till Victor Sellstedt

File: en/contact/body.md [hunk 5]
```diff
@@ -1,63 +1,60 @@
...
-## Näringslivskontakt
+## Business relations
 
-Näringslivsgruppen har hand om sektionens kontakter med företag. Oavsett
-anledning är du alltid välkommen att kontakta oss:
+The industry group handles the chapter's contacts with companies. You can always contact us at:
 
-**Näringslivsgruppen**<br />
-E-post: [foretag@d.kth.se](mailto:foretag@d.kth.se)
+**The industry group**<br />
```
Comment (bullfest) [inline en/contact/body.md]: På alla andra platser används business relations för näringsliv, kan vara bra att vara konsekvent

**Comment by bullfest** [comment]
@LudvigJaniuk It's better to actually do it right when doing the translation rather than doing a bad job that has to be redone.

**Comment by LudwikJaniuk** [comment]
@bullfest I agree that we should get the translations improved as soon as possible.

**Pull request merged**